### PR TITLE
docs - add content for PXF JDBC connector

### PR DESCRIPTION
**Pull request by lisakowen**
in this PR:
- create new PXF topic "Accessing an External SQL Database with PXF (JDBC)"
   - treat it like topics for other connectors
   - use some content from hawq docs and the connector README
   - create a simple postgresql example (previous example was mysql)
- misc updates on other pages with references to the connectors and pushdown

**plan to create a partitioning-related example in a follow on PR/release 

doc review site link to new topic:
- http://docs-lisa-pxfjdbc.cfapps.io/600/pxf/jdbc_pxf.html

### CODE REVIEW

**Comment by lisakowen** [comment]
@lavjain - please pass along the review request to other team members as you see fit.

**Comment by lisakowen** [comment]
@leskin-in would you be interested in reviewing?

File: gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+6. Restart PXF on *each* Greenplum Database segment host with the following command:
+
+    ``` shell
+    gpadmin@gpmaster$ gpssh -e -v -f seghostfile "/usr/local/greenplum-db/pxf/bin/pxf restart"
+    ```
+
+#### <a id="ex_readjdbc"></a>Read from the PostgreSQL Table
+
+Perform the following procedure to create a PXF external table referencing the `forpxf_table1` PostgreSQL table that you created in the previous section, and read the data in the table:
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb]: referencing -> that references

**Comment by lavjain** [comment]
@lisakowen I haven't used the JDBC profile in a while. Adding @kongyew, @denalex  and @shivzone

File: gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Some of your data may already reside in an external SQL database. PXF provides access to this data via the PXF JDBC Connector. The JDBC connector is a JDBC client. It can read data from and write data to SQL databases including MySQL, ORACLE, PostgreSQL, and Hive.
+
+This section describes how to use the PXF JDBC connector to access data in an external SQL database, including how to create and query or insert data into a PXF external table that references a table in an external database.
+
+**Note**: The JDBC Connector does not guarantee consistency when writing to an external SQL database. Be aware that even if an `INSERT` operation fails, some data maybe still be written to the external database table. If you require consistency for writes, consider writing to a staging table in the external database, and loading to the target table only after verifying the write operation.
```
Comment (leskin-in) [inline gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb]: As for me, `Be aware that if an `INSERT` operation fails, some data may be written to the external database table.` sounds better.

File: gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+To access data in an external SQL database, you create a readable or writable Greenplum Database external table that references the external database table. The Greenplum and external database tables must have the same definition; the column names, types, and order must match.
+
+Use the following syntax to create a Greenplum Database external table that references an external SQL database table and uses the JDBC connector to read or write data:
+
+``` sql
+CREATE [READABLE | WRITABLE] EXTERNAL TABLE <table_name>
+    ( <column_name> <data_type> [, ...] | LIKE <other_table> )
+LOCATION ('pxf://<external-table-name>?PROFILE=Jdbc[&<custom-option>=<value>[...]]')
+FORMAT 'CUSTOM' (FORMATTER='pxfwritable_import'|'pxfwritable_import');
```
Comment (leskin-in) [inline gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb]: Probably the correct text is `FORMATTER='pxfwritable_import'|'pxfwritable_export'`?

File: gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+| BATCH_SIZE | Write | Integer identifying the number of `INSERT` operations to batch to the external SQL database. PXF always validates a `BATCH_SIZE` option, even when provided on a read operation. Batching is enabled by default. |
+| POOL_SIZE | Write | Enable thread pooling on `INSERT` operations and identify the number of threads in the pool. Thread pooling is disabled by default. |
+| PARTITION_BY | Read | The partition column, \<column-name\>:\<column-type\>. You may specify only one partition column. The JDBC connector supports `date`, `int`, and `enum` \<column-type\> values. A null `PARTITION_BY` defaults to a single fragment. |
+| RANGE | Read | (Used only when `PARTITION_BY` is specified.) The query range, \<start-value\>[:\<end-value\>]. When the partition column is an `enum` type, `RANGE` must specify a list of values, each of which forms its own fragment. If the partition column is an `int` or `date` type, `RANGE` must specify a finite left-closed range. That is, the range includes the \<start-value\> but does *not* include the \<end-value\>. If the partition column is a `date` type, use the `yyyy-MM-dd` date format. |
```
Comment (leskin-in) [inline gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb]: I propose we explicitly state that `RANGE` is **required** when `PARTITION_BY` is present. `Required when `PARTITION_BY` is specified.` will do, I think.

**Comment by leskin-in** [comment]
Good job @lisakowen ! The doc file is detailed and covers all the aspects of PXF JDBC connector usage. A new example with PostgreSQL is also good.

**Comment by lisakowen** [comment]
thank you for reviewing, @leskin-in !  i have pushed your requested edits.

**Comment by lisakowen** [comment]
thank you also for reviewing, @dyozie and @divyabhargov .

**Pull request merged**